deployments/prod/environment.py

```diff
@@ -86,4 +86,9 @@ def env() -> Mapping[str, Optional[str]]:
         # FIXME: Unset once the client ID becomes optional
         #        https://github.com/DataBiosphere/azul/issues/3088
         'AZUL_GOOGLE_OAUTH2_CLIENT_ID': '473200283737-h5e1l7neunbuesrtgjf8b12lb7o3jf1m.apps.googleusercontent.com',
+
+        'AZUL_INDEXER_CONCURRENCY': '300',
```

Comment:
As specified in the ticket, only the `contribute` lambda's concurrency should be increased. This will require creating a second variable for the aggregation lambda.

I think we should do this as a backwards-compatibility breaking config change:

Reply:
```suggestion
        'AZUL_CONTRIBUTION_CONCURRENCY': '300/64',
        'AZUL_AGGREGATION_CONCURRENCY': '64'
```

The number after the slash is for the retry lambda. `42` is equivalent to `42/42` and `42/` should be considered invalid.